18F/identity-idp
# LG-11906 Update Review Step UI with Selfie to Match Document Step with Selfie

## 🎫 Ticket

https://cm-jira.usa.gov/browse/LG-11906

## 🛠 Summary of changes

This PR extracts the two components Daweis created for the `documents-step` and adds them to the `review-step`.

A little more explanation since this is confusing:
- When a user uploads documents they're on the `documents-step`.
- If they upload a failing (blurry, etc) picture and submit.
- They land on a very similar page (with front/back/selfie  uploads)
- But that's actually not the same `documents-step`. It's the `review-step` which is basically the same, but also handles the errors from the earlier submission.

## 📜 Testing Plan
⚠️ Notes in [this thread](https://gsa-tts.slack.com/archives/C056RD1NEHW/p1703273361661079) on enabling the selfie feature as of 12/22/2023  ⚠️  Turning on the featureflag doesn't show the selfie box right now. Testing workaround is in the first step.
- With the selfie feature flag on. Or, you can also force `getSelfieCaptureEnabled` to return `true` as a work around.
- Go to the Front/back selfie upload page.
- You should see the numbered headings and selfie box per [this PR](https://github.com/18F/identity-idp/pull/9746). Take a screenshot of this page for comparing to a later step.
- Upload one of the yaml files that cause the submission to fail for all three fields: front, back, and selfie.
- Click "Submit"
- Click the button to try again online.
- The page you see now is the `review-step` page. The numbered headers and selfie step should look the same as your screenshot from earlier.

## 👀 Screenshots

If relevant, include a screenshot or screen capture of the changes.

<details>
<summary>Before:</summary>

</details>

<details>
<summary>After:</summary>

</details>
-->
